Sources/RegexBuilder/CharacterClass.swift

```diff
@@ -72,27 +98,58 @@ extension RegexComponent where Self == CharacterClass {
     ]))
   }
 
-  public static var horizontalWhitespace: CharacterClass {
-    .init(unconverted: .horizontalWhitespace)
+  /// A character class that matches any element that is a "word character".
```

Comment:
Just to double check, is there any better description than "word character"? "Word character" can be mentioned as an aside but that's more of a historical note. @Azoy does Unicode have another name?